tests/repoFindByDate.py

```diff
@@ -158,7 +158,7 @@ def map_cfg(self, dataId, write):
         lookups = self.storage.lookup(lookupProperties='date', reference=None,
                                      dataId=dataId, template=template)
         lookups.sort()
-        if len(lookups) is not 0:
+        if len(lookups):
```

Comment:
> I realize that 0 is a singleton in Python but that's not generally how tests against numbers are done.


oh right, I stumble on that sometimes; the `is not` operator is incorrect even if it does work. I think `len(lookups) != 0` is probably the most expressive. `if lookups` is probably the most pythonic(?)
I'm ok with either, but I like the former better.